src/templates/partials/header.html

```diff
@@ -1,15 +1,13 @@
 <h1 class="site">
```

Comment:
what do you think about ```s/partials/_includes``` since Nunjucks calls it ```{% include %}```, and to be consistent with fireplace?